asserts/asserts_test.go

```diff
@@ -56,34 +59,36 @@ func (as *assertsSuite) TestDecodeEmptyBodyAllDefaults(c *C) {
 	c.Check(a.AuthorityID(), Equals, "auth-id1")
 }
 
+const exampleEmptyBody2NlNl = "type: test-only\n" +
+	"authority-id: auth-id1\n" +
+	"revision: 0\n" +
+	"body-length: 0" +
+	"\n\n" +
+	"\n\n" +
+	"openpgp c2ln\n"
```

Comment:
I think it would help to be consistent with the trailing "\n" on these consts, i.e. exampleEmptyBodyAllDefaults has no trailing "\n" but the other 2 (?) do.